logic/src/commonTest/kotlin/com/wire/kalium/logic/data/register/RegisterAccountRepositoryTest.kt

```diff
@@ -109,16 +112,22 @@ class RegisterAccountRepositoryTest {
         val code = "123456"
         val name = "user_name"
         val password = "password"
-        val expected = RegisterResponse(
-            id = "user_id",
+        val expected = UserDTO(
+            id = UserId("user_uuid", "domain.com"),
```

Comment:
The previous value is `user_id` and the new one is `user_uuid`, is that correct?

Reply:
This changes nothing, it's just some value for test it can be anything, but I will fix it anyway